File: JAVA/S6_LINKEDLIST/L1_1DLINKEDLIST/_1_list.java



class Node{
    int data;
    Node next;
    Node(int data){
        this.data=data;
        this.next=null;
    }
}
class LinkedList{
    Node head;
    void insertbeg(int data){
        Node newnode=new Node(data);
        newnode.next=head;
        head=newnode;
    }
    void display(){
        Node temp=head;
        while(temp!=null){
            System.out.print(temp.data+" ");
            temp=temp.next;
        }

    }
}
public class _1_list{
    public static void main(String[] args){
        Node head=null;
        LinkedList list=new LinkedList();
        list.insertbeg(23);
        list.insertbeg(45);
        list.display();

    }
}

File: python/S6_LINKEDLIST/L1_1DLINKEDLIST/1_list.py
class Node:
    def __init__(self,data):
        self.next=None
        self.data=data
class List:
    def __init__(self):
        self.head=None
    def insertbeg(self,data):
        newnode=Node(data)
        newnode.next=self.head
        self.head=newnode
    def display(self):
        temp=self.head
        while temp is not None:
            print(temp.data,end=" ")
            temp=temp.next
if __name__=="__main__":
    l=List()
    l.insertbeg(34)
    l.insertbeg(23)
    l.display()


        